libr/core/rvc.c

```diff
@@ -730,6 +732,47 @@ R_API bool r_vc_commit(const char *rp, const char *message, const char *author,
 		eprintf ("Can't commit");
 		return false;
 	}
+	if (!message) {
+#if __WINDOWS__ //fuck you
+	char *whoami = r_sys_whoami ();
+	if (!whoami) {
+		return false;
+	}
+	char *path = r_str_newf ("c:\\\\users\\%s\\appdata\\local\\tmp\\rvcmessage",
```

Comment:
Dont hardcode such paths there are rutil apis for that

Reply:
What exactly do want me to use